python/foglamp/services/core/api/common.py

```diff
@@ -41,6 +43,15 @@
            curl -X GET http://localhost:8081/foglamp/ping
     """
 
+    try:
+        auth_token = request.token
+    except AttributeError:
+        cfg_mgr = ConfigurationManager(connect.get_storage())
+        category_item = await cfg_mgr.get_category_item('rest_api', 'allowPing')
+        allow_ping = True if category_item['value'] == 'true' else False
```

Comment:
Configuration manager value saved as it is for below scenarios:
curl -X PUT -d '{"value": "True"}' http://localhost:8081/foglamp/category/rest_api/allowPing
curl -X PUT -d '{"value": "TRUE"}' http://localhost:8081/foglamp/category/rest_api/allowPing

So we should handle `category_item['value'].lower()`

Reply:
done